### PR TITLE
Updated main.yml

### PR DESCRIPTION
**Pull request by Janvi-Thakkar**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -45,11 +52,11 @@ jobs:
         with:
           name: build
           path: ./build
-      - name: Deploy to Firebase
+    
+      - name: GitHub Action for Firebase
         uses: Niweera/firebase-action@master
```
[the remark this answers is not in the document]
Comment (Niweera) [reply .github/workflows/main.yml:57]: sure you can use it.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -45,11 +52,11 @@ jobs:
...
-      - name: Deploy to Firebase
-        uses: Niweera/firebase-action@master
+    
+      - name: GitHub Action for Firebase
+        uses: w9jds/firebase-action@master
         with:
           args: deploy --only hosting
         env:
           FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
-          PROJECT_ID: ${{ secrets.REACT_APP_FIREBASE_PROJECT_ID }}
```
Comment (Niweera) [inline .github/workflows/main.yml:54]: sorry i missed this the last time, why did you remove the project ID?

Comment (Janvi-Thakkar) [reply .github/workflows/main.yml:54]: it is not a required field if we have already mentioned our project id in .firebaserc

Comment (Niweera) [reply .github/workflows/main.yml:54]: For now let's keep it there.

**Review by Niweera — APPROVED** [review]

**Review by Sarfraz-droid — APPROVED** [review]
